distributed/http/scheduler/prometheus/core.py

```diff
@@ -146,15 +146,11 @@ def collect(self) -> Iterator[GaugeMetricFamily | CounterMetricFamily]:
                 prefix_state_counts.add_metric([tp.name, state], count)
         yield prefix_state_counts
 
-        now = time()
-        max_tick_duration = max(
-            self.server.digests_max["tick_duration"],
-            now - self.server._last_tick,
-        )
+        observations = self.server._observed_tick_durations or [(None, 0)]
         yield GaugeMetricFamily(
             self.build_name("tick_duration_maximum_seconds"),
             "Maximum tick duration observed since Prometheus last scraped metrics",
```

Comment:
```suggestion
            self.build_name("rolling_tick_duration_maximum_seconds"),
            "Rolling maximum tick duration",
```
possible adding the actual window size.